### PR TITLE
Added navigation over stages of SetPollingSource editing wizard

### PR DESCRIPTION
**Pull request by dmitriy-borzenko**

### CODE REVIEW

File: src/app/dataset-view/additional-components/metadata-component/components/add-polling-source/steps/preprocess-step/preprocess-step.component.ts [hunk 1]
```diff
@@ -55,7 +57,9 @@ export class PreprocessStepComponent extends BaseComponent implements OnInit {
     }
 
     private initDefaultQueriesSection(query = ""): void {
+        console.log("2");
```
Comment (zaychenko-sergei) [inline src/app/dataset-view/additional-components/metadata-component/components/add-polling-source/steps/preprocess-step/preprocess-step.component.ts:60]: Clean this

File: src/app/dataset-view/additional-components/metadata-component/components/add-polling-source/steps/preprocess-step/preprocess-step.component.ts [hunk 2]
```diff
@@ -55,7 +57,9 @@ export class PreprocessStepComponent extends BaseComponent implements OnInit {
...
         if (!this.preprocessValue.queries.length) {
+            console.log("3");
```
Comment (zaychenko-sergei) [inline src/app/dataset-view/additional-components/metadata-component/components/add-polling-source/steps/preprocess-step/preprocess-step.component.ts:62]: and this

File: src/app/dataset-view/additional-components/metadata-component/components/add-polling-source/steps/preprocess-step/preprocess-step.component.ts (hunk 1 above)
Comment (dmitriy-borzenko) [reply src/app/dataset-view/additional-components/metadata-component/components/add-polling-source/steps/preprocess-step/preprocess-step.component.ts:60]: Fixed

File: src/app/dataset-view/additional-components/metadata-component/components/add-polling-source/steps/preprocess-step/preprocess-step.component.ts (hunk 2 above)
Comment (dmitriy-borzenko) [reply src/app/dataset-view/additional-components/metadata-component/components/add-polling-source/steps/preprocess-step/preprocess-step.component.ts:62]: Fixed

**Pull request merged**